spec/features/venues/view_venue_details_spec.rb

```diff
@@ -21,4 +21,20 @@
     expect(page).to have_content(venue.address)
     expect(page).to have_content(venue.capacity)
   end
+
+  scenario 'If T accessible, user should see message: T is nearby' do
+    venue = FactoryGirl.create(:venue, t_accessible: true)
+
+    visit venue_path(venue)
+
+    expect(page).to have_content(venue.display_t_accessible)
```

Comment:
Ok. We went with the `let` statement at the top to allow for later flexibility, but make sure we are testing that the string actually shows up. This seems like a good idea, especially if we add that logic back in to our view.